### PR TITLE
blog overview

### PR DESCRIPTION
**Pull request by sugandhadev**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -69,6 +72,7 @@ import { GlobalErrorHandler } from './services/shared/globalErrorHandler'
         { path: 'termsandconditions', component: TermsAndConditionsComponent, pathMatch: 'full'},
         { path: 'privacypolicy', component: PrivacyComponent, pathMatch: 'full'},
         { path: 'feedback', component: FeedbackComponent, pathMatch: 'full'},
+        { path: 'blogs', component: BlogsOverviewComponent, pathMatch: 'full'},
```
Comment (shreyagarwaldev) [inline src/app/app.module.ts]: change it to "blog"

File: src/app/app.module.ts [hunk 2]
```diff
@@ -69,6 +74,8 @@ import { GlobalErrorHandler } from './services/shared/globalErrorHandler'
         { path: 'termsandconditions', component: TermsAndConditionsComponent, pathMatch: 'full'},
         { path: 'privacypolicy', component: PrivacyComponent, pathMatch: 'full'},
         { path: 'feedback', component: FeedbackComponent, pathMatch: 'full'},
+        { path: 'photography-blogs', component: BlogsOverviewComponent, pathMatch: 'full'},
```
Comment (shreyagarwaldev) [inline src/app/app.module.ts]: nope, just "blog"

File: src/app/blog/blog.component.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+@Component({
+    templateUrl: './blog.component.html',
+    styleUrls: ['./blog.component.scss']
+})
+
+export class BlogComponent {
+
+    blogId: string;
+    // @ViewChild(WorkshopsListComponent) workshopsListChildComp:WorkshopsListComponent;
```
Comment (shreyagarwaldev) [inline src/app/blog/blog.component.ts]: remove line

File: src/app/blog/blog.component.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    blog: IBlogDetail;
+
+    constructor(private blogRepository: BlogRepository,
+        private globalConstantsRepository: GlobalConstantsRepository,
+        private route: ActivatedRoute,
+        gaService: GoogleAnalyticsService,
+        title: Title,
+        meta: Meta) {
+        gaService.trackPageView('blogs');
+        title.setTitle('Photography blogs - Pixelated Planet')
```
Comment (shreyagarwaldev) [inline src/app/blog/blog.component.ts]: change to title of blog. Also change other meta tag properties to correct values

File: src/app/navbar/navbar.component.scss [hunk 5]
```diff
@@ -140,6 +140,10 @@ nav {
         font-family: 'Comfortaa', cursive;
     }
 }
+// .nav.navbar-nav>li>a {
+//     padding-left: 8px;
+//     padding-right: 8px;
+// }
```
Comment (shreyagarwaldev) [inline src/app/navbar/navbar.component.scss]: remove the whole chunk

File: src/app/workshops/workshops.component.html [hunk 6]
```diff
@@ -5,7 +5,7 @@
             (locationFilterChanged)="setLocationIdList($event)" (categoryFilterChanged)="setCategoryList($event)" (applyFilters)="toggleFilterDropdown($event)"></workshop-filter>
     </div>
     <div class="list">
-        <workshops-list [activePage]="pageNumber"></workshops-list>
+        <workshops-list [activePage]="pageNumber" [itemLimit]="0"></workshops-list>
```
Comment (shreyagarwaldev) [inline src/app/workshops/workshops.component.html]: 0?

**Review by shreyagarwaldev — CHANGES_REQUESTED** [review]

File: src/app/workshops/workshops.component.html (hunk 6 above)
Comment (sugandhadev) [reply src/app/workshops/workshops.component.html]: yep.. this is a required property and we want to show the page numbers for all

Comment (sugandhadev) [reply src/app/workshops/workshops.component.html]: moved it to the script

**Review by shreyagarwaldev — APPROVED** [review]

**Pull request merged**